research-software-directory/RSD-as-a-service
# 888 project overview

# Project overview page - v2

Closes #888
Closes #896 

Changes proposed in this pull request:
* Project overview page is redesigned to be in line with new software overview page
* Mouse-over message for counts included in the cards on project and software overview page
* restyled keyword "chip" in cards: smaller fonts, capitalized, one line "clamp" and bit more padding
* removed old filter components
* removed old filter RPC's from database scripts
* moved "old" software and project cards  to "components/user/.." folder. "Old" cards are still used in user and organisation sections
* changed background color
* changed avatar background color based on lighthouse contrast warning
* order on start date is descending (newest first)
* order on end date is ascending (oldest first)
* labels changed: View page, Edit page to View Project/Software, Edit Project/Software
* button "Clear" type/color changed to primary

How to test:
* `make start` to rebuild app and generate data
*  navigate to project overview page
*  test order, filters and search options
*  validate mouse over messages on the impact and output counts in the card
* login as rsd_admin to be able to edit existing project items.
* edit project images using `fit image` flag (see image 3). Try card without image too.

## Example grid view
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/2ed6ace5-d732-435e-a7bb-1243241d301f)

## Example list view
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/474fb7ef-eb32-4799-b18e-d09e599c9b53)

## Project card image format (fit image flag)
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/34798586-a330-4ec0-8ee2-821c63bc58ce)


PR Checklist:
